packages/clay-list/src/QuickActionMenu.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import ClayIcon from '@clayui/icon';
+import React from 'react';
+
+interface ItemProps
+	extends React.HTMLAttributes<HTMLAnchorElement | HTMLSpanElement> {
+	href?: string;
+	spritemap?: string;
+	symbol: string;
+}
+
+const QuickActionMenuItem: React.FunctionComponent<ItemProps> = ({
+	children,
+	className,
+	href,
+	spritemap,
+	symbol,
+	...otherProps
+}) => {
+	const ElementTag = href ? 'a' : 'button';
```

Comment:
Main reason I didn't use ClayLink or ClayButton is because we don't use the class names associated with them. If we needed the class names and functionality from those components I would be up for using them, but I think its a stretch to use them anywhere we use an anchor or button element.

One option for this specific instance though would be to create another component that would conditionally render an anchor or button.

Reply:
No problem, maybe it would be interesting to have a component to check conditionally, maybe even to use for other use cases...